accounts/swagger/connection.py

```diff
@@ -36,6 +36,6 @@
     responses={200: openapi.Response("Realtive updated successfully")},
 )(update_connection)
 
-connections_endpoint = swagger_auto_schema(method="Get", responses={200: openapi.Response("Account logged out sucessfully")})(
-    connectios
-)
+retrieve_connections_endpoint = swagger_auto_schema(
+    method="Get", responses={200: openapi.Response("Connections retrieved sucessfully")}
+)(connections)
```

Comment:
* We don't need to create a swagger function for a `GET` request, so I guess we have to delete it, did you do it for a reason?